docs/coordinates/representations.rst

```diff
@@ -212,18 +212,123 @@ To see how the operations work, consider the following examples::
         (  3.06161700e-16,  -5.00000000e+00,  12.),
         ( -4.00000000e+00,  -3.00000000e+00, -12.)]]>
 
+For coordinates, one also has to deal with proper motions, which describe
```

Comment:
(As with the two docstrings above, I pushed up some changes to this section that I think are controversial.)